api/src/api/users/user_routes.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from flask import Response
+
+from src.api import response
+from src.api.route_utils import raise_flask_error
+from src.api.users import user_schemas
+from src.api.users.user_blueprint import user_blueprint
+from src.auth.api_key_auth import api_key_auth
+
+logger = logging.getLogger(__name__)
+
+# Descriptions in OpenAPI support markdown https://swagger.io/specification/
+SHARED_ALPHA_DESCRIPTION = """
+__ALPHA VERSION__
+
+This endpoint in its current form is primarily for testing and feedback.
+
+Features in this endpoint are still under heavy development, and subject to change. Not for production use.
+
+See [Release Phases](https://github.com/github/roadmap?tab=readme-ov-file#release-phases) for further details.
+"""
+
+
+@user_blueprint.post("/user/token")
+@user_blueprint.input(
+    user_schemas.UserTokenHeaderSchema, location="headers", arg_name="x_oauth_login_gov"
+)
+@user_blueprint.output(user_schemas.UserTokenResponseSchema)
+@user_blueprint.auth_required(api_key_auth)
+@user_blueprint.doc(  # should we include this?
+    description=SHARED_ALPHA_DESCRIPTION, responses=[200, 400]
+)
+def user_token(x_oauth_login_gov: str) -> response.ApiResponse | Response:
+    logger.info("POST /v1/users/user/token")
+
+    if x_oauth_login_gov:
+        data = {
+            "token": "the token goes here!",
+            "user": {
+                "user_id": "abc-...",
+                "email": "example@gmail.com",
+                "external_user_type": "login_gov",
+            },
+            "is_user_new": True,
+        }
+        return response.ApiResponse(message="Success", data=data)
+
+    message = "Missing X-OAuth-login-gov header"
+    logger.error(message)
```

Comment:
Note - even once built out, we don't want to log an error.

To a user it is an error sure, but error logs on our side would trip alerts. A user calling our endpoint and hitting a 4xx is not something we want to alert.

We only want to log non-info if it is bad for the system.
```suggestion
    logger.info(message)
```

Reply:
Makes sense!